utils/BaseDetector.py

```diff
@@ -31,20 +31,21 @@ def feedCap(self, im, draw=True):
             'frame': None,
             'faces': None,
             'list_of_ids': None,
-            'face_bboxes': []
+            'face_bboxes': [],
+            'current_ids': None
         }
         self.frameCounter += 1
         if draw:
             print("drawing")
-            im, faces, face_bboxes = update_tracker(self, im, draw, self.targetTrackId)
+            im, faces, face_bboxes, current_ids,lost = update_tracker(self, im, draw, self.targetTrackId)
```

Comment:
`current_ids` to store the object being tracked in current frame. this is needed is to update `Detector` state, which is used in subsequent `det.feedcap(im)`